x-pack/packages/ml/aiops_log_rate_analysis/api/stream_reducer.ts

```diff
@@ -80,7 +85,12 @@ export const logRateAnalysisResultsSlice = createSlice({
     resetGroups: (state) => {
       state.significantItemsGroups = [];
     },
-    resetAll: () => getDefaultState(),
+    // reset the results but keeps
```

Comment:
Nit: unfinished comment explanation?

Reply:
Thanks for spotting this! Fixed in [6941aa7](https://github.com/elastic/kibana/pull/188648/commits/6941aa72590acfd86cb2d8b1e6b5a1189a17abd5).